edx_sga/__init__.py

```diff
@@ -2,4 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = "0.18.0"
+__version__ = "0.19.0"
```

Comment:
small note: you don't need to bump the version. Our release bot will take care of it.

Reply:
Gotcha, fixed.